### PR TITLE
Fix reboot

### PR DESCRIPTION
**Pull request by obino**
It's been reported that under certain circumstances, the system won't come up properly after a reboot.

### CODE REVIEW

**Comment by obino** [comment]
ocd run https://ocd.appscale.com:8080/job/Daily%20Build/593

File: AppController/djinn.rb [hunk 1]
```diff
@@ -804,7 +808,22 @@ def kill(secret)
...
+        # Let's stop all other nodes.
+        threads << Thread.new {
+          @nodes.each { |node|
+            if node.private_ip != my_node.private_ip
+              acc = AppControllerClient.new(ip, @@secret)
+              begin
+                acc.kill()
+                Djinn.log_info("kill: sent kill command to node at #{ip}")
+              rescue FailedNodeException
+                Djinn.log_warn("kill: failed to talk to node at #{ip} while")
```
Comment (menivaitsi) [inline AppController/djinn.rb]: please finish the log sentence and add '.'

File: AppController/djinn.rb [hunk 2]
```diff
@@ -804,7 +808,22 @@ def kill(secret)
...
 
       if my_node.is_login?
+        # Let's stop all other nodes.
+        threads << Thread.new {
+          @nodes.each { |node|
+            if node.private_ip != my_node.private_ip
+              acc = AppControllerClient.new(ip, @@secret)
+              begin
+                acc.kill()
+                Djinn.log_info("kill: sent kill command to node at #{ip}")
```
Comment (menivaitsi) [inline AppController/djinn.rb]: Add '.' at the end,

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/600/

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/602/

**Pull request merged**